### PR TITLE
move the errors html to a partial and reuse it where needed

### PR DESCRIPTION
**Pull request by wilburpowery**
It's a lot nicer that way if you want to change the structure or layout of the errors. Just change the partial. 😁

Also used a @forelse loop on the `users.profile` view to remove some @if checks. If they were there for a specific reason, feel free to revert back to the original. 🤝

### CODE REVIEW

File: resources/views/_partials/errors.blade.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@if ($errors->any())
```
Comment (ludo237) [inline resources/views/_partials/errors.blade.php]: Is there any particular standard that says that partials should be inside a `_folder` ?

Comment (assertchris) [reply resources/views/_partials/errors.blade.php]: Don't really like that style either.

Comment (ludo237) [reply resources/views/_partials/errors.blade.php]: I think `_partials` folder it too generic.

Probably a structure like:
`views/components/form/error.blade.php` allows for a better component structures

For example. Let's say the project needs two kind of view for form errors one with classic list, one with inline list. It's easy to manage with a dedicated directory of **form components**

What do you think?

Comment (assertchris) [reply resources/views/_partials/errors.blade.php]: I think that depends too much on what design this gets to matter right now.

**Comment by assertchris** [comment]
I think this might just be failing because of the encryption stuff in `.travis.yml`. I'm happy to merge it if it's mostly ok.

**Comment by assertchris** [comment]
@wilburpowery thank you :)

**Pull request merged**